purchase_order_no_owner/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,12 @@
+This module adds no_owner field in the purchase order,
+allowing users to indicate that the ordered inventory is not associated with any specific owner.
```

Comment:
More emphasis should be put on preventing the oversight than giving options to users.